<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>This file is part of the xTuple ERP: PostBooks Edition, a free and
open source Enterprise Resource Planning software suite,
Copyright (c) 1999-2014 by OpenMFG LLC, d/b/a xTuple.
It is licensed to you under the Common Public Attribution License
version 1.0, the full text of which (including xTuple-specific Exhibits)
is available at www.xtuple.com/CPAL.  By using this software, you agree
to be bound by its terms.</comment>
 <class>scrapTrans</class>
 <widget class="QWidget" name="scrapTrans">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scrap Transaction</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="_transDateLit">
     <property name="text">
      <string>Transaction Date:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="4">
    <widget class="DLineEdit" name="_transDate"/>
   </item>
   <item row="0" column="5" colspan="3">
    <widget class="QLabel" name="_usernameLit">
     <property name="text">
      <string>Username:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="8">
    <widget class="QLabel" name="_username">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="10" rowspan="2" colspan="2">
    <layout class="QVBoxLayout">
     <item>
      <widget class="QPushButton" name="_close">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="_post">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Post</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="9">
    <widget class="ItemCluster" name="_item"/>
   </item>
   <item row="2" column="4">
    <widget class="QLabel" name="_beforeLit">
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>32767</height>
      </size>
     </property>
     <property name="text">
      <string>Before</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="6" colspan="2">
    <widget class="QLabel" name="_afterLit">
     <property name="text">
      <string>After</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="_warehouseLit">
     <property name="text">
      <string>&amp;Site:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>_warehouse</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="4" colspan="2">
    <widget class="XLabel" name="_beforeQty">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="6" colspan="3">
    <widget class="XLabel" name="_afterQty">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="_qtyLit">
     <property name="text">
      <string>Scrap Qty.:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="1" rowspan="2" colspan="3">
    <widget class="XLineEdit" name="_qty">
     <property name="alignment">
      <set>Qt::AlignRight</set>
     </property>
    </widget>
   </item>
   <item row="6" column="0" rowspan="2">
    <widget class="QLabel" name="_documentNumLit">
     <property name="text">
      <string>Document #:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="10">
    <widget class="XLineEdit" name="_documentNum"/>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="_notesLit">
     <property name="text">
      <string>Notes:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="12">
    <widget class="XTextEdit" name="_notes">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="spellEnable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="WComboBox" name="_warehouse"/>
   </item>
   <item row="0" column="9">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>47</width>
       <height>65</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DLineEdit</class>
   <extends>QWidget</extends>
   <header>datecluster.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ItemCluster</class>
   <extends>QWidget</extends>
   <header>itemcluster.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WComboBox</class>
   <extends>XComboBox</extends>
   <header>wcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>XComboBox</class>
   <extends>QComboBox</extends>
   <header>xcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>XLabel</class>
   <extends>QLabel</extends>
   <header>xlabel.h</header>
  </customwidget>
  <customwidget>
   <class>XLineEdit</class>
   <extends>QLineEdit</extends>
   <header>xlineedit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>XTextEdit</class>
   <extends>QTextEdit</extends>
   <header>xtextedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>_transDate</tabstop>
  <tabstop>_item</tabstop>
  <tabstop>_warehouse</tabstop>
  <tabstop>_qty</tabstop>
  <tabstop>_documentNum</tabstop>
  <tabstop>_notes</tabstop>
  <tabstop>_post</tabstop>
  <tabstop>_close</tabstop>
 </tabstops>
 <includes>
  <include location="global">parameter.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>_item</sender>
   <signal>valid(bool)</signal>
   <receiver>_post</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_close</sender>
   <signal>clicked()</signal>
   <receiver>scrapTrans</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_item</sender>
   <signal>newId(int)</signal>
   <receiver>_warehouse</receiver>
   <slot>findItemsites(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_item</sender>
   <signal>warehouseIdChanged(int)</signal>
   <receiver>_warehouse</receiver>
   <slot>setId(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
